client/starwhale/base/client/models/models.py

```diff
@@ -1480,6 +1480,26 @@ class ServiceSpec(SwBaseModel):
     apis: Optional[List[ApiSpec]] = None
 
 
+class OptionType(SwBaseModel):
+    name: str
+    param_type: str
+    # for Choice type
+    choices: Optional[List[str]] = None
+    case_sensitive: bool = False
+
+
+class OptionField(SwBaseModel):
+    name: str
+    opts: List[str]
```

Comment:
Maybe we can remove `opts` field, only use `name` field as input argument name.

Reply:
removed